src/ReleaseHistory.md

```diff
@@ -1,44 +1,45 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
 ## **v2.1.17** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.1.17) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.1.17) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.1.17) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.1.17)
-* FEATURE: Introduce SarifConsolidator to shrink large log files. https://github.com/microsoft/sarif-sdk/issues/1675
-* BUGFIX: Analysis rule SARIF1017 incorrectly rejected index-valued properties that referred to taxonomies. https://github.com/microsoft/sarif-sdk/issues/1678
 * API NON-BREAKING: emit all core object model members as 'virtual'.
+* FEATURE: Introduce SarifConsolidator to shrink large log files. [#1675](https://github.com/microsoft/sarif-sdk/issues/1675)
+* BUGFIX: Analysis rule SARIF1017 incorrectly rejected index-valued properties that referred to taxonomies. [#1678](https://github.com/microsoft/sarif-sdk/issues/1678)
+* BUGFIX: `match-results-forward-command` dropped log contents and mishandled `rules` array. [#1684](https://github.com/microsoft/sarif-sdk/issues/1684)
```

Comment:
>BUGFIX [](start = 2, length = 6)

This is the new line. The rest of it is just me improving the Markdown by making the links more compact, _e.g._, `[[[https://github.com/microsoft/sarif-sdk/issues/1575`](https://github.com/microsoft/sarif-sdk/issues/1575`)](https://github.com/microsoft/sarif-sdk/issues/1575`)](https://github.com/microsoft/sarif-sdk/issues/1575`) => `[#1575]([[[https://github.com/microsoft/sarif-sdk/issues/1575)`](https://github.com/microsoft/sarif-sdk/issues/1575)`)](https://github.com/microsoft/sarif-sdk/issues/1575)`)](https://github.com/microsoft/sarif-sdk/issues/1575)`)

Reply:
Just curious, why did you stop this scrub where you did? Did you just exhaust your 'opportunistinc improvements' capital? should we continue this clean-up as each of us modifies this file? (a reasonable approach)

---
In reply to: [328351950](https://github.com/microsoft/sarif-sdk/pull/1686#discussion_r328351950) [](ancestors = 328351950)

Reply:
Exactly that. I stopped at the point where I thought "Michael's going to wonder how much time I spent on this" :-) Answer: a few minutes. Happy to finish it if you want!

---
In reply to: [328778424](https://github.com/microsoft/sarif-sdk/pull/1686#discussion_r328778424) [](ancestors = 328778424,328351950)

Reply:
This file changes all the time, I suggest we use the 'waves on the beach' approach and fix it over time. The deserialization issues continue to ring in my head as an alarm bell. :)

---
In reply to: [328780288](https://github.com/microsoft/sarif-sdk/pull/1686#discussion_r328780288) [](ancestors = 328780288,328778424,328351950)

Reply:
Agreed.

---
In reply to: [328784413](https://github.com/microsoft/sarif-sdk/pull/1686#discussion_r328784413) [](ancestors = 328784413,328780288,328778424,328351950)